doc/user/content/overview/arangements.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Arrangements"
+description: "Understand how Materialize arrangements work."
+menu:
+  main:
+    parent: 'overview'
+    weight: 4
+---
+
+We call the mechanisms that maintain materialized views for Materialize dataflows **arrangements**, and understanding them better can help you make decisions that will reduce memory usage while maintaining performance.
+
+<!-- trade off memory and performance -->
+
+## What are arrangements?
+
+Arrangements are streams of update triples composed of `(data, time, diff)`. `data` is structured as a key-value pair, and arrangements are indexed on the `data` keys. An arrangement records all updates, but an index only maintains the current accumulation of `diff` for each `(data, time)` pair.
```

Comment:
This is the first place where "index" is introduced, and I'm having trouble understanding what you're referring to.  Are you intending to compare this to the index used in a traditional database?

Related, wdyt of adding something like:
> Arrangements are the streaming dataflow equivalent of relational database indexes.

Reply:
Yeah, sorry, that wasn't ready for review yet.